plugins/text-freetype2/text-freetype2.c

```diff
@@ -130,6 +130,16 @@ static obs_properties_t *ft2_source_properties(void *unused)
 	obs_properties_add_text(props, "text",
 		obs_module_text("Text"), OBS_TEXT_MULTILINE);
 
+	obs_property_t *alignmentProp = obs_properties_add_list(props,
+	                "alignment", obs_module_text("Alignment"),
```

Comment:
Style:  It seems the example Jim gave you used two tabs to indent here, but you used spaces.  Please switch to two tabs here for now.

Reply:
According to [coding guidelines](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst#coding-guidelines) I should use `tabs for indentation and spaces for alignment`. But I can change all to tabs if you want.

Reply:
There's not really a reason to align like this here though.  You would align things like sequential variable declarations or maybe complex multi-line if conditions (`text-freetype2.c:379`, `libobs/obs.c:980`, `libobs/obs.c:1200`).  The sole exception in your examples is `libobs/obs.c:968`, and I can't really speak to that one.  Jim provided an example that used tabs, and the project normally uses tabs for the situation where you've used spaces.